tensorflow_datasets/text/disco_fuse.py

```diff
@@ -41,54 +43,55 @@ def __init__(self, **kwargs):
     """
     super(DiscoFuseConfig, self).__init__(**kwargs)
 
+
 class DiscoFuse(tfds.core.GeneratorBasedBuilder):
   """DiscoFuse: A Large-Scale Dataset for Discourse-Based Sentence Fusion"""
 
   VERSION = tfds.core.Version('0.1.0')
   BUILDER_CONFIGS = [
-    DiscoFuseConfig(
-      name="wikipedia",
-      version="0.0.1",
-      description="DiscoFuse from Wikipedia Articles",
-    ),
-    DiscoFuseConfig(
-      name="sports",
-      version="0.0.1",
-      description="Sports from Sports Articles",
-    )]
+      DiscoFuseConfig(
```

Comment:
Indentation should be 2. Check TensorFlow code style [link](https://www.tensorflow.org/community/contribute/code_style).

Reply:
If I add two spaces the lint keeps asking me to delete it

Reply:
use 
```python3 
# pylint: disable=bad-indentation
```

Reply:
Sorry, but I think this is perfectly fine. See [here](https://github.com/tensorflow/datasets/blob/07ae48d19be4b5692a7b01eed11f6ce23c8a5db3/tensorflow_datasets/image/lsun.py#L70).

Reply:
So, should I change it?

Reply:
No changes required. @us any suggestions ?

Reply:
The indentation is correct:
2 space for code blocks (`if`, `class`, `def`, `with`, `for`,...)
4 spaces for `[]`, `()`, `{}`